### ISSUE TITLE
Add Organic Maps to Taginfo Projects

### ISSUE THREAD

**Issue by Dimitar5555** [organicmaps/organicmaps#3917]
See https://wiki.openstreetmap.org/wiki/Taginfo/Projects and https://github.com/taginfo/taginfo-projects/blob/master/CONTRIBUTING.md for more information

**Comment by matkoniecz** [comment]
Maybe it could be done in parts? For example "Organic Maps Editor POIs" as one of segments?

**Comment by matkoniecz** [comment]
@biodranik Would it be acceptable to add one more step to https://github.com/organicmaps/organicmaps/blob/17229123415bec80c331e301524d797ec19e98c9/docs/RELEASE_MANAGEMENT.md ?

Or have such script running and committing on its own after something changed?

Or should it stay manual with human running script and providing PR?

Or should it be a separate repository?

**Comment by biodranik** [comment]
Which step?

**Comment by matkoniecz** [comment]
Updating and commiting tag usage files for taginfo.

**Comment by biodranik** [comment]
I see, then Organic Maps will be listed in "Projects" tab on taginfo, right?

Is it enough to store this file in our Github, and periodically regenerate it/commit updates to it?

**Comment by matkoniecz** [comment]
You are right on both of them (#4977 waiting to be reviewed is the first step to this and https://github.com/organicmaps/organicmaps/issues/4895#issuecomment-1492600624 ).

**Comment by dvdmrtnz** [comment]
Organic Maps is now live on the taginfo website! 🥳

https://taginfo.openstreetmap.org/projects/organicmaps
